packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporterNodeBase.ts

```diff
@@ -27,22 +30,30 @@ export abstract class CollectorExporterNodeBase<
   ExportItem,
   ServiceRequest
 > extends CollectorExporterBase<
-  CollectorExporterConfigBase,
+  CollectorExporterNodeConfigBase,
   ExportItem,
   ServiceRequest
 > {
   DEFAULT_HEADERS: Record<string, string> = {};
   headers: Record<string, string>;
-  constructor(config: CollectorExporterConfigBase = {}) {
+  keepAlive: boolean = true;
+  httpAgentOptions: http.AgentOptions | https.AgentOptions = {};
+  constructor(config: CollectorExporterNodeConfigBase = {}) {
     super(config);
     if ((config as any).metadata) {
       this.logger.warn('Metadata cannot be set when using http');
     }
     this.headers =
       parseHeaders(config.headers, this.logger) || this.DEFAULT_HEADERS;
+    if (config.keepAlive !== undefined && !config.keepAlive) {
+      this.keepAlive = config.keepAlive;
+    }
+    if (config.httpAgentOptions) {
+      this.httpAgentOptions = config.httpAgentOptions;
```

Comment:
to prevent mutating the entered value later as this is an object
```suggestion
      this.httpAgentOptions = Object.assign({}, config.httpAgentOptions);
```